models/src/it/java/io/sphere/sdk/reviews/ReviewIntegrationTest.java

```diff
@@ -0,0 +1,392 @@
+    }
+
+    @Test
+    public void testGetById() throws Exception {
+        withProduct(client(), product ->
+                withCustomer(client(), customer ->
+                        withReview(client(), product, customer, review -> {
+                            PagedQueryResult<Review> result = client().execute(ReviewQuery.of().byId(review.getId()));
+                            Optional<Review> optReview =
+                                    result.getResults().stream().filter(review1 -> review1.getId().equals(review.getId())).findFirst();
+                            assertTrue(optReview.isPresent());
+                            assertEquals(optReview.get(), review);
+                        })));
+    }
+
+    @Test
+    public void testGetByProductId() throws Exception {
+        withProduct(client(), product ->
+                withCustomer(client(), customer ->
+                        withReview(client(), product, customer, review -> {
+                            PagedQueryResult<Review> result = client().execute(ReviewQuery.of().byProductId(review.getProductId()));
+                            Optional<Review> optReview =
+                                    result.getResults().stream().filter(review1 -> review1.getId().equals(review.getId())).findFirst();
+                            assertTrue(optReview.isPresent());
+                            assertEquals(optReview.get(), review);
+                        })));
+    }
+
+    @Test
+    public void testFilterProductIdAndCustomerId() throws Exception {
+        withProduct(client(), product ->
+                withCustomer(client(), customer ->
+                        withReview(client(), product, customer, review -> {
+                            PagedQueryResult<Review> result = client().execute(ReviewQuery.of()
+                                    .withPredicate(Predicate.of("customerId = \"" + review.getCustomerId() + "\"")));
+                            Optional<Review> optReview =
+                                    result.getResults().stream().filter(review1 -> review1.getId().equals(review.getId())).findFirst();
+                            assertTrue(optReview.isPresent());
+                            assertEquals(optReview.get(), review);
+
+                            result = client().execute(ReviewQuery.of()
+                                    .withPredicate(Predicate.of("productId = \"" + review.getProductId() + "\"")));
+                            optReview = result.getResults().stream().filter(review1 -> review1.getId().equals(review.getId())).findFirst();
+                            assertTrue(optReview.isPresent());
+                            assertEquals(optReview.get(), review);
+
+                            result = client().execute(ReviewQuery.of()
+                                    .withPredicate(Predicate.of("productId = \"" + review.getProductId() + "\" and customerId = \"" + review.getCustomerId() + "\"")));
```

Comment:
I t would be nice if you could here use ReviewQueryModel instead of String concatenation.